SimG4CMS/ShowerLibraryProducer/plugins/HcalForwardLibWriter.cc

```diff
@@ -97,10 +99,15 @@ void HcalForwardLibWriter::analyze(const edm::Event& iEvent, const edm::EventSet
       ngood++;
       if (ngood > nshowers)
         continue;
+      unsigned int s = nphot;  //++
```

Comment:
@bsunanda , better not to use the name of variable "s", there may be clash with CLHEP.